packages/game-app/src/_shared/index.ts

```diff
@@ -2,9 +2,13 @@ import { buildStore } from './store';
 import CONFIG from '@pipeline/app-config';
 import firebase from 'firebase/app';
 import 'firebase/analytics';
+import 'firebase/firestore';
+import 'firebase/functions';
```

Comment:
why?

Reply:
Isn't it because of the emulators?

Reply:
yes excacly, i'll try to understand the impact on the main bundle size and try to find a workaround.